### PR TITLE
(feat): Weekly Newsletter 2016-07-01

### PR DESCRIPTION
**Pull request by thomasem**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-557cb990c2/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-1544201cc6/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-0d2218c185/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-abb1073052/)

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Weekly news: DockerCon 2016, Docker 1.12.0, and Cluster Build Issue"
```
Comment (kmsholcomb) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: ... cluster build issue

(Using sentence-style case for headings.)

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+date: 2016-07-01
+comments: true
+author: Thomas Maddox <thomas.maddox@rackspace.com>
+published: true
+excerpt: >
+  We had an awsome time at DockerCon 2016. Our booth was buzzing with discussions daily about the future of Carina and Docker. We've also been working behind the scenes looking at Docker Swarm Mode support which will be included in the Docker 1.12.0 release. Other than that, we resolved a cluster build issue that came up and users shouldn't be having trouble creating clusters anymore.
```
Comment (kmsholcomb) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: ... awesome ...

... support, which will be ...

... we resolved a cluster build issues, and users shouldn't have any more trouble creating clusters.

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+![Screenshot of DockerCon]({% asset_path "2016-06-03-weekly-news-pycon-and-release-information/DockerCon.png" %})
+
+We had a great time at [DockerCon](http://2016.dockercon.com/) this year in Seattle. We enjoyed discussing Carina with the steady flow of folks coming by to see what this Carina thing was and how to use it. Thank you all for coming by and catching up on Carina. And congratulations to the winner of our Parrot Bebop Quadcopter from the contest at the booth!
+
+### Docker 1.12.0 Swarm Mode
+
+One of the many topics at DockerCon 2016 was around the new Swarm Mode in Docker 1.12.0, where the Swarm services come with the Docker daemon. We are looking at supporting this in the future, so stay tuned!
+
+### Resolved Erroring Clusters
```
Comment (kmsholcomb) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: Resolved erroring clusters

(sentence-style case)

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+On Thursday evening we began experiencing new clusters immediately going to "error" on create. This [issue](https://status.getcarina.com/incidents/f00dpl1vkzm9) was resolved ~30 minutes later and you should no longer see this problem. If you do see any problems come up, please let us know!
```
Comment (kmsholcomb) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: On Thursday evening, we began experiencing an issue with new clusters ...

... later, and you ...

... If you do see any problems, please let us know!

**Comment by kmsholcomb** [comment]
Hi @thomasem. I left just a few copy-editing comments in the file. Thanks!

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md (hunk 2 above)
Comment (thomasem) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: Done, done, and done.

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md (hunk 3 above)
Comment (thomasem) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: Done.

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md (hunk 4 above)
Comment (thomasem) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: Done, done, and done.

**Comment by thomasem** [comment]
@kmsholcomb Thank you! I fixed it up according to your suggestions.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-3293164e4a/)

File: _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md (hunk 1 above)
Comment (thomasem) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: Oh, whoops. Good catch!

Comment (thomasem) [inline _posts/2016-07-01-weekly-news-dockercon2016-docker_swarm_mode-cluster-build-issue.md]: Done.

**Comment by everett-toews** [comment]
👍

**Pull request merged**